### DISCUSSION TITLE
Asklist from Amara

### DISCUSSION THREAD

**Discussion by T99** [frc1711/raptors1711.com#21] [Ideas]
- Super Science Saturday page
 - Sponsorships/Donations page
   - Direct link to TCAPS donation page.
 - Upcoming Events page
   - Volunteering events
   - Practices
   - Competitions
 - Contact page

**Comment by T99** [comment]
Stretch goal: Instagram-style person tagging and links to a 'meet this person' page.

**Reply by Dr-WaSaBi** [reply]
With most being minors, can we do that without giving out too much info about the students??  Are there privacy issues with doing this is what I'm trying to say.
